apps/dash-3d-image-partitioning/plotly-common/README.md

```diff
@@ -0,0 +1,16 @@
+Stuff I use with `plotly.py` and `dash` webapps.
```

Comment:
I think these instructions are not necessary since the files are included in the PR?